ocs_ci/ocs/resources/ocs.py

```diff
@@ -335,3 +335,13 @@ def ocs_install_verification(timeout=600, skip_osd_distribution_check=False):
             assert not node_names.count(node) > 1, (
                 "OSD's are not distributed evenly across worker nodes"
             )
+
+    # Verify node and provisioner secret names in storage class
+    log.info("Verifying node and provisioner secret names in storage class.")
+    sc_rbd = storage_class.get(resource_name='ocs-storagecluster-ceph-rbd')
+    sc_cephfs = storage_class.get(resource_name='ocs-storagecluster-cephfs')
+    assert sc_rbd['parameters']['csi.storage.k8s.io/node-stage-secret-name'] == 'rook-csi-rbd-node'
+    assert sc_rbd['parameters']['csi.storage.k8s.io/provisioner-secret-name'] == 'rook-csi-rbd-provisioner'
```

Comment:
same for all below 4 checks, needs to be from constant,  I am curious why we need this?

Reply:
Done.
This check is to ensure that there are two different secrets for provisioning and mount. Earlier both were same which will give additional permission to nodes where volume is mounted. This was fixed and we need to ensure it is available.
More info : 
https://bugzilla.redhat.com/show_bug.cgi?id=1760604#c0
https://bugzilla.redhat.com/show_bug.cgi?id=1760604#c4